.circleci/config.yml

```diff
@@ -17,6 +17,14 @@ defaults:
         [ -n "$COVERAGE" -a "$CIRCLE_BRANCH" != release -a -z "$CIRCLE_TAG" ] && CMAKE_OPTIONS="$CMAKE_OPTIONS -DCOVERAGE=ON"
         cmake .. -DCMAKE_BUILD_TYPE=Release $CMAKE_OPTIONS
         make -j4
+  - run_build_ossfuzz: &run_build_ossfuzz
```

Comment:
If it is only used once, it does not need to go here actually, but still fine.

Reply:
Will let it stay because we may want to consider another fuzz CI for regressions that reuses this stub.